src/components/Button/index.js

```diff
@@ -120,14 +120,17 @@ const propTypes = {
 
     /** A ref to forward the button */
     forwardedRef: refPropTypes,
+
+    ...withThemePropTypes,
+    ...withThemeStylesPropTypes,
 };
 
 const defaultProps = {
     text: '',
     shouldShowRightIcon: false,
     icon: null,
     iconRight: Expensicons.ArrowRight,
-    iconFill: themeColors.textLight,
+    iconFill: undefined,
```

Comment:
why undefined?

Reply:
because we can't use the dynamic color from the `withTheme` HOC outside of the React lifecycle. We can set this to either dark or light theme colors instead

Reply:
gotcha, that makes sense